dev_guide/deployments.adoc

```diff
@@ -273,6 +273,14 @@ image stream changes and the new tag value differs from the current image
 specified in the deployment configuration's `helloworld` container, a new
 deployment is created using the new tag value for the `helloworld` container.
 
+The following table explains the possible combinations between ConfigChange
+and ImageChange triggers.
+
+  Trigger                  | true                                      | false
+  -----------------------  | ----------------------------------------- | ------------------------------------------------------
+  ConfigChange             | Deployments are started automatically     | Deployments need to be started manually
+  ImageChange (automatic)  | Images are always resolved automatically  | Only the initial image will be resolved in the template
```

Comment:
@rhcarvalho: (sorry, I was out last week) my issue is the opposite: the images
are already on the node (they aren't built, so openshift doesn't "know" about
them) and the only thing I need is for the deployment to start when the DC is
created.  If I only have the ConfigChange trigger, that's what happens.  But
when I add an ImageChangeTrigger, I get this on the log:

    Couldn't create initial deployment for deploymentConfig "x/y": DeploymentConfig "y" is invalid: triggers[1].imageChange.tag: Invalid value: "latest": no image recorded for x/y:latest